feature-dapp-impl/src/main/java/io/novafoundation/nova/feature_dapp_impl/presentation/main/DappCategoriesAdapter.kt

```diff
@@ -60,18 +65,27 @@ private object DappDiffCallback : DiffUtil.ItemCallback<DAppCategoryModel>() {
 
 class DappCategoryViewHolder(
     override val containerView: View,
+    private val imageLoader: ImageLoader,
     private val itemHandler: DappCategoriesAdapter.Handler,
 ) : RecyclerView.ViewHolder(containerView), LayoutContainer {
 
     fun bind(item: DAppCategoryModel) = with(containerView) {
-        itemDappCategory.text = item.name
+        itemDappCategoryIcon.loadOrHide(item.iconUrl, imageLoader)
+        itemDappCategoryText.text = item.name
 
         bindSelected(item.selected)
 
         containerView.setOnClickListener { itemHandler.onCategoryClicked(item.id) }
     }
 
-    fun bindSelected(isSelected: Boolean) = with(containerView) {
-        itemDappCategory.isSelected = isSelected
+    fun bindSelected(isSelected: Boolean) {
+        itemView.isSelected = isSelected
+
+        // We must set tint to image view programmatically since we can't specify the state for default color
```

Comment:
Can you please explain that? Why cant we use a default state in selector drawable?

Reply:
Yes, we can't set default color for state list, since icons have different colors but the color in state list is constant